cluster/bin/load-snappy-env.sh

```diff
@@ -32,6 +32,18 @@ if [ -z "$SNAPPY_ENV_LOADED" ]; then
   # Returns the parent of the directory this script lives in.
   parent_dir="`absPath "$FWDIR/.."`"
 
+  if [ -z "$MALLOC_ARENA_MAX" ]; then
+    export MALLOC_ARENA_MAX=4
+  fi
+
+  if [ -z "$MALLOC_MMAP_THRESHOLD_" ]; then
+    export MALLOC_MMAP_THRESHOLD_=131072
+  fi
+
+  if [ -z "$MALLOC_MMAP_MAX_" ]; then
+    export MALLOC_MMAP_MAX_=2147483647
+  fi
+
```

Comment:
Better to move this to snappy-env.sh rather than load-snappy-env.sh. Also it looks like the bin/snappy script loads only spark-env.sh and not snappy-env.sh. Can you update the snappy script to also load snappy-env.sh? Also moving to spark-env.sh will be better so that the same settings apply to Spark clusters too.

Reply:
From yesterday's observation, it should work as it is. I have added loading of snappy env in snappy script too.